src/content/recipes/styled-components.md

```diff
@@ -220,10 +266,10 @@ export const parameters = {
   },
 };
 
-export const decorators = [withTheme];
+export const decorators = [withGlobalStyle, withTheme];
 ```
 
-Now, components made with Styled Components will get the theme through the `theme` prop. Let's update the example components to use the theme.
+Now, components made with styled-components will get the theme through the `theme` prop. Let's update the example components to use the theme.
```

Comment:
Oh, in conjunction with \<previous comment>, we could add an aside after this paragraph, saying that this includes the GlobalStyle component?

Reply:
Addressed this [here](https://github.com/storybookjs/frontpage/pull/455/commits/c1fbd29d84c350bac9e519f9c6f09bc635e56c91). What do you think?

Reply:
That's a good addition, but not quite what I was referring to here.

Reply:
Chatted to KG in Slack, going to ignore this